src/rpc/misc.cpp

```diff
@@ -484,24 +487,26 @@ UniValue setmocktime(const UniValue& params, bool fHelp)
     return NullUniValue;
 }
 
-bool getAddressFromIndex(const int &type, const uint160 &hash, std::string &address)
+bool getAddressFromIndex(const AddressType &type, const uint160 &hash, std::string &address)
```

Comment:
I believe it does not quite make sense to pass `type` (which is an `int`) as a reference in this case. I'd suggest
```suggestion
bool getAddressFromIndex(const AddressType type, const uint160 &hash, std::string &address)
```

Reply:
Done